packages/main/src/plugin/color-registry.ts

```diff
@@ -543,4 +544,18 @@ export class ColorRegistry {
       light: colorPalette.gray[500],
     });
   }
+
+  // links
+  protected initLink(): void {
+    const sNav = 'link';
+
+    this.registerColor(`${sNav}`, {
+      dark: colorPalette.purple[400],
+      light: colorPalette.purple[700],
+    });
+    this.registerColor(`${sNav}-bg`, {
```

Comment:
I think `hover` should appear in the name

```suggestion
    this.registerColor(`${sNav}-hover-bg`, {
```